api/src/config.rs

```diff
@@ -681,10 +681,10 @@ impl CacheConfigV2 {
         }
 
         if self.prefetch.enable {
-            if self.prefetch.batch_size > 0x10000000 {
+            if self.prefetch.prefetch_request_batch_size > 0x10000000 {
```

Comment:
how about `prefetch_batch_size` instead?

Reply:
There's at least three types of `batch_size` in nydus. Is it explicit enough to leave it as `batch_size`?

Reply:
Reverted to `prefetch.batch_size`. Please reply again if there's some other problems.